test/hound_ci.rb

```diff
@@ -0,0 +1,2 @@
+puts "double quotes"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.